doc/dts/device_tree.rst

```diff
@@ -0,0 +1,208 @@
+.. _device-tree:
+
+Device Tree in Zephyr
+########################
+
+Introduction to Device Tree
+***************************
+
+Device tree is a way of describing hardware and configuration information
```

Comment:
Please add a note making it clear that Zephyr does not use the Blob itself at runtime